app/questionnaire/return_location.py

```diff
@@ -0,0 +1,27 @@
+from dataclasses import asdict, dataclass
+
+
+@dataclass(kw_only=True, frozen=True)
+class ReturnLocation:
+    """
+    Used to store return locations in the questionnaire.
+
+    return_to: The name of the section to return to
```

Comment:
Not sure this is correct that it is the name of the section, the return_to is often a summary page or a calculated summary page etc

Reply:
Ah, that's a good point, I will update it to `The name of the summary page to return to` which hopefully catches each name?

Reply:
I think the 'type' of summary page might be clearer

Reply:
Great, thanks both 👍